### PR TITLE
Move manager permissions CTA

### PR DESCRIPTION
**Pull request by flvndvd**
## Description

This PR moves the "Manage permissions" CTA to a less engaging place.

Base on figma designs from [here](https://www.figma.com/file/QOxHwppG64GtPocpDhS6Y7/Product?type=design&node-id=2391-31273&mode=design&t=Y3d5xDNnxXY0o3jY-0).

image

### CODE REVIEW

**Review by Duncid — APPROVED** [review]

**Comment by spolu** [comment]
wait wait how does it work when the two are merged like in your screenshot?

**Comment by spolu** [comment]
Ok your screenshot is a lie based on my tests :)

Only the solo manage permissions button is moved right?

File: front/pages/w/[wId]/builder/data-sources/[name]/index.tsx [hunk 1]
```diff
@@ -789,6 +789,13 @@ function ManagedDataSourceView({
     };
   };
 
+  const shouldDisplayManagePermissionButton = [
```
[the remark this answers is not in the document]
Comment (flvndvd) [reply front/pages/w/[wId]/builder/data-sources/[name]/index.tsx:792]: I'll patch things up.

**Comment by flvndvd** [comment]
> Member

Yes I only add notion connected locally, so I had to tweak the include.

File: front/pages/w/[wId]/builder/data-sources/[name]/index.tsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (spolu) [reply front/pages/w/[wId]/builder/data-sources/[name]/index.tsx:792]: Thanks thanks :pray:

**Pull request merged**